contrib/pg_tde/README.md

```diff
@@ -1,4 +1,5 @@
 [![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/percona/pg_tde/badge)](https://scorecard.dev/viewer/?uri=github.com/percona/pg_tde)
+[![codecov](https://codecov.io/github/percona/postgres/graph/badge.svg?token=Wow78BMYdP)](https://codecov.io/github/percona/postgres)
```

Comment:
While it makes more sense to keep it here, this is also more hidden, I'm not sure if it's a good idea or bad.

Reply:
Then we can keep in main readme but mention that it's just tde coverage `pg_tde coverage: <badge>`

Reply:
How about this? 
<img width="678" alt="Screenshot 2025-04-18 at 19 07 06" src="https://github.com/user-attachments/assets/9b563340-e76f-4bfc-8994-058f00fa3fc9" />

Reply:
Something like `Code Coverage (pg_tde): <Badge> ` or `Code Coverage for pg_tde: <Badge>` will look better. But not a big deal. :)

Reply:
Changed
<img width="522" alt="Screenshot 2025-04-22 at 16 12 27" src="https://github.com/user-attachments/assets/1d03a705-0e72-40cc-a7ca-315b15b26b46" />